CMakeLists.txt

```diff
@@ -32,6 +35,12 @@ option(UNICORN_INSTALL "Enable unicorn installation" ${PROJECT_IS_TOP_LEVEL})
 set(UNICORN_ARCH "x86;arm;aarch64;riscv;mips;sparc;m68k;ppc;s390x" CACHE STRING "Enabled unicorn architectures")
 option(UNICORN_TRACER "Trace unicorn execution" OFF)
 
+if(DEFINED ENV{CARGO})
```

Comment:
Is there any specific reason not to put this to build.rs?

Reply:
i've pushed another commit which just uses CARGO flag (that is set automatically by cargo) to control the msvcrt linking.

Reply:
i've updated it to not need CARGO indicator at all now